src/components/TokenBalance.test.tsx

```diff
@@ -713,109 +721,3 @@ describe.each([
     expect(getElementText(tree.getByTestId('TotalTokenBalance'))).toEqual('₱-')
   })
 })
-
-describe('renders the network icon on the home screen to differentiate between ETH on Ethereum, Arbitrum, and Optimism', () => {
```

Comment:
these tests were superfluous, as it was testing the underlying behavior of TokenIcon component using HomeTokenBalance, just updated a test in FiatExchangeTokenBalance to ensure TokenIcon is displayed when there's a single token with balance as HomeTokenBalance is going to be removed as part of the drawer nav cleanup